ci-app/app-repo/cloudbuild.yaml

```diff
@@ -29,6 +29,6 @@ steps:
                   && kpt fn source constraints/ hydrated-manifests/ > hydrated-manifests/kpt-manifests.yaml']
 - id: 'Validate against policies'
   # This step validates that all resources comply with all policies.
-  name: 'gcr.io/config-management-release/policy-controller-validate'
+  name: 'gcr.io/kpt-fn/gatekeeper'
```

Comment:
I'm wondering here if we should go with an even more generic way to use `kpt` here?

Like this:
```
name: 'gcr.io/kpt-dev/kpt'
  entrypoint: '/bin/sh'
  args: ['-c', 'kpt fn eval hydrated-manifests/kpt-manifests.yaml --image gcr.io/kpt-fn/gatekeeper:v0.2']
```
The idea is that this command could be taken as-is in any other CI/CD tools like Jenkins, locally, etc.

Reply:
my update was for a different reason but you bring up a good point, we need to make sure that this works with the other ci tools you mentioned.